.github/workflows/benchmark.yml

```diff
@@ -28,7 +28,7 @@ jobs:
 
       - uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # pin@v5
         with:
-          go-version: '1.21'
+          go-version-file: 'go.mod'
```

Comment:
you could cache dependencies with:
```suggestion
          go-version-file: 'go.mod'
          cache-dependency-path: go.sum
```
Take into account, that's the minimal version to use but not the maximum as stated in their [doc](https://github.com/actions/setup-go?tab=readme-ov-file#getting-go-version-from-the-gomod-file)

Reply:
I think it caches by default, no? The docs seems to imply that you need this option in case you have the go.sum elsewhere in the repo (not at root).
![Fri Mar 28 02:22:00 PM UTC 2025](https://github.com/user-attachments/assets/f9924091-e9ff-406b-961a-1e38789c1ec4)

or am I wrong?